Source/Core/Core/DSP/DSPIntCCUtil.cpp

```diff
@@ -106,44 +106,39 @@ void Update_SR_LZ(bool value)
 		g_dsp.r.sr &= ~SR_LOGIC_ZERO;
 }
 
-inline int GetMultiplyModifier()
+static bool IsCarry()
 {
-	return (g_dsp.r.sr & SR_MUL_MODIFY)?1:2;
+	return (g_dsp.r.sr & SR_CARRY) != 0;
```

Comment:
This will cause warnings in clang, since it doesn't result in a boolean expression, but one of type int.

e.g. http://ideone.com/pYNkRF